index.html

```diff
@@ -20,6 +20,9 @@
             
             START !
         </div></a>
+        <a href="rules.html"><div class="btn-secondary">
+            Rules
```

Comment:
Pages looks good.
One Suggestion .
On "Rules" button add some border radius property so that it starts matching with the "Start" button on the homepage.
Otherwise looks good!

Reply:
added , good suggestion.